eos-tech-support/eos-write-installer

```diff
@@ -97,7 +97,7 @@ dependencies=(
     [dd]='coreutils'
     [gzip]='gzip'
     [mkfs.exfat]='exfat-utils'
-    [pv]='pv'
+    [xz]='xz-utils'
```

Comment:
It was already checking for gzip rather than zcat so I thought I'd be consistent. Have changed them all so we're instead correct, rather than merely consistent.